pkg/kubectl/cmd/util/factory_builder.go

```diff
@@ -48,7 +48,7 @@ func NewBuilderFactory(clientAccessFactory ClientAccessFactory, objectMappingFac
 	return f
 }
 
-func (f *ring2Factory) PrinterForCommand(cmd *cobra.Command, isLocal bool, options printers.PrintOptions) (printers.ResourcePrinter, error) {
+func (f *ring2Factory) PrinterForCommand(isLocal bool, options printers.PrintOptions) (printers.ResourcePrinter, error) {
```

Comment:
@fabianofranz given that this method would no longer depend on `*cobra.Command` with the proposed changes, I think it would make sense to rename this method or combine it with another printing method on the factory. WDYT?